packages/main/src/plugin/util/manifest.ts

```diff
@@ -39,13 +39,48 @@ const GUESSED_MANIFEST_SIZE = 50 * KB;
 //
 // We will check this within ImageInfo
 export function guessIsManifest(image: ImageInfo, connectionType: string): boolean {
+  // There is an odd case that if the image has been renamed, we may not be able to safely detect
```

Comment:
Just wondering why we are not relying on the Names attributes:

```
C:\Users\Jeff\apps>podman image ls --format json
[
    {
        "Id": "00339ea1c0d1525fd93fc3aa9c926c75ad2ec32a15627c32bd3924a7dbe667cd",
        "ParentId": "",
        "RepoTags": null,
        "RepoDigests": [
            "localhost/mymanifest123@sha256:20b959ad5960230b65a77b746bdbf5d991ade4d7a129c2554e167acdcc990531",
            "localhost/mymanifest@sha256:20b959ad5960230b65a77b746bdbf5d991ade4d7a129c2554e167acdcc990531"
        ],
        "Size": 110,
        "SharedSize": 0,
        "VirtualSize": 110,
        "Labels": null,
        "Containers": 0,
        "Names": [
            "localhost/mymanifest123:latest",
            "localhost/mymanifest:latest"
        ],
        "Digest": "sha256:20b959ad5960230b65a77b746bdbf5d991ade4d7a129c2554e167acdcc990531",
        "History": [
            "localhost/mymanifest:latest",
            "localhost/mymanifest123:latest"
        ],
        "Created": 1713253600,
        "CreatedAt": "2024-04-16T07:46:40Z"
    },
```

Reply:
Hi! We do not use names as they can still be changed and do not contain the digest value. We use digests to more accurate detect if it's been renamed or not.